src/Google.Api.Gax/PageStreamer.cs

```diff
@@ -66,19 +69,23 @@ public sealed class PageStreamer<TResource, TRequest, TResponse, TToken>
         /// <summary>
         /// Lazily fetches resources a page at a time.
         /// </summary>
+        /// <param name="callSettings">The <see cref="CallSettings"/> to apply to each RPC invocation.</param>
         /// <param name="initialRequest">The initial request to send. If this contains a page token,
         /// that token is maintained. This is also passed to the request provider along with a token
         /// to provide another request. (The provider may build a new object, or modify the existing request.)</param>
         /// <param name="responseFetcher">A function to fetch a response containing a page of results given a request.</param>
-        /// <returns>A sequence of resources, which are fetched a page at a time. Must not be null.</returns>
-        public IEnumerable<TResource> Fetch(TRequest initialRequest, Func<TRequest, TResponse> responseFetcher)
+        /// <returns>A sequence of resources, which are fetched a page at a time. Will not be null.</returns>
+        public IEnumerable<TResource> Fetch(
+            CallSettings callSettings,
+            TRequest initialRequest,
+            ApiCall<TRequest, TResponse>.SyncCall responseFetcher)
```

Comment:
It's quite tempting to make both methods just take an `ApiCall<TRequest, Response>`...